src/fsharp/FSharp.Core/set.fsi

```diff
@@ -22,7 +22,13 @@ type Set<[<EqualityConditionalOn>]'T  when 'T : comparison> =
     ///
     /// <returns>The result set.</returns>
     /// 
-    /// <example-tbd></example-tbd>
+    /// <example id="set-new">
+    /// <code lang="fsharp">
+    /// let seq = seq { 1 .. 3 }
+    /// Set(seq)
```

Comment:
Maybe add print here too, as in other examples? So if people copy-paste code to the script/repl, they get some friendly output.

Reply:
Will do.